packages/browser/src/core/inspector/index.ts

```diff
@@ -0,0 +1,9 @@
+import { Inspector } from 'segment-inspector-core'
```

Comment:
I think any isomorphism would be better baked into `segment-inspector-core` (core is the keyword), rather than the responsibility of the consumer

Reply:
I don't follow. Can you elaborate?

Reply:
```suggestion
import type { Inspector } from 'segment-inspector-core'

```

Reply:
@zikaari ignore the isomorphism bit, realizing that the Inspector was just a type changed my p-o-v =)

Reply:
> I don't follow. Can you elaborate?

I changed my mind =)